rewrite-maven/src/main/java/org/openrewrite/maven/tree/ResolvedPom.java

```diff
@@ -325,6 +325,9 @@ public String getPackaging() {
             } else {
```

Comment:
Turns out the `propVal` just above here is already a dead store in the original, so I'll refactor a little to clean that out as well.